package.json

```diff
@@ -21,23 +21,30 @@
   ],
   "repository": "Saevon/bytes.js",
   "devDependencies": {
+    "@types/mocha": "8.0.4",
+    "@types/node": "14.14.10",
     "eslint": "5.13.0",
     "mocha": "5.2.0",
-    "nyc": "13.2.0"
+    "nyc": "13.2.0",
+    "ts-node": "9.1.0",
+    "tsconfig-paths": "3.9.0",
+    "typescript": "4.1.2"
   },
   "files": [
     "History.md",
     "LICENSE",
     "Readme.md",
-    "index.js"
+    "index.js",
+    "index.d.ts"
   ],
   "engines": {
     "node": ">= 0.8"
   },
   "scripts": {
     "lint": "eslint .",
-    "test": "mocha --check-leaks --reporter spec",
+    "test": "mocha test/* --check-leaks --reporter spec --require ts-node/register --require tsconfig-paths/register",
     "test-ci": "nyc --reporter=text npm test",
-    "test-cov": "nyc --reporter=html --reporter=text npm test"
+    "test-cov": "nyc --reporter=html --reporter=text npm test",
+    "typecheck": "tsc -b"
```

Comment:
Is this a helper?  ideally there should only be one test command for us to run

Reply:
I added it just for completeness, not out of necessity. You would only run this to make double sure. Mocha handles type errors just fine from what I can see though, so I can omit this if you prefer.